src/styles/getTheme.js

```diff
@@ -82,6 +82,9 @@ export default function getTheme(theme, ...more) {
                 borderRadius: (spacing.actionButtonSize - 16) / 2,
                 backgroundColor: grey500,
             },
+            speedDialActionLabel: {
+                color: grey600,
```

Comment:
Hey @NewOldMax :) We should probably use `palette.secondaryTextColor`. What do you think?